tests/network/vmi_lifecycle.go

```diff
@@ -48,7 +48,7 @@ import (
 	"kubevirt.io/kubevirt/tests/testsuite"
 )
 
-var _ = SIGDescribe("[crit:high][arm64][vendor:cnv-qe@redhat.com][level:component]", func() {
+var _ = SIGDescribe("[crit:high][arm64][vendor:cnv-qe@redhat.com][level:component]", decorators.WgS390x, func() {
```

Comment:
nit - do you also need to add [s390x] to the description here? That said I'm slightly confused why we are hard coding arches into these strings If I'm honest.